tests/drivers/gpio/gpio_nrf/src/main.c

```diff
@@ -16,6 +16,8 @@
 #error Unsupported board
 #endif
 
+const struct device *port = DEVICE_DT_GET(TEST_NODE);
```

Comment:
This is done both here and in each testcase.

Reply:
I have added setup but it was denied, see above: 'Test setup() function is not related to the added test.'

Reply:
Not sure what you mean. My point is that it's done multiple times, which is unnecessary.

Reply:
Ok sorry i see the doubled code now.

Reply:
Updated